packages/core/src/index.ts

```diff
@@ -34,3 +34,11 @@ export {
 } from 'next/document'
 
 export {default as dynamic} from 'next/dynamic'
+
+export {Error} from 'next/error'
+
+export {AppProps} from 'next'
+
+export {InferGetStaticPropsType} from 'next'
+
+export {InferGetServerSidePropsType} from 'next'
```

Comment:
Great, looking good!

One small change that would be good is to move these exports to around line 15 where other items are being exported from `next`. That way `from 'next'` only appears in this file once. Make sense?

Reply:
yeah make sense, got it :) will get this done, thank you!